### ISSUE TITLE
hyper-parameters about AVA experiment

### ISSUE THREAD

**Issue by adwardlee** [facebookresearch/SlowFast#41]
Hi,

Thanks for sharing the code. I notice that the hyper-parameters in the paper are different with the config files in the codebase (e.g. learning rate schedule), so I suppose the AVA hyper-parameters may be also different. If possible, could you please provide the hyper-parameters to train the AVA pipeline, especially for small amount of GPUs to reproduce the results in the paper? Thanks.

**Comment by takatosp1** [comment]
Thanks a lot for using the code. We will release the AVA pipeline with the configs soon.

**Comment by takatosp1** [comment]
Could you clarify what do you mean by hyper-parameters are different?
If you are referring about the learning rate, the lr is for single machine in the config we released. If you want to play with multiple machines, you need to scale the lr following with linear scaling rule.

**Comment by adwardlee** [comment]
Sorry, I didnot notice the codebase has been updated and the base_lr 0.4 is deleted. Thanks for you clarification.
